content/en/docs/test.md

```diff
@@ -369,9 +369,13 @@ Notes catch the reader's attention without a sense of urgency.
 
 You can have multiple paragraphs and block-level elements inside an admonition.
 
-| Or | a | table |
+| Or | | Even |
+| :---------------- | ----------------: | --------------: |
```

Comment:
```suggestion
|:---:|:---:|:-----:|
```

Reply:
I made the columns in two different directions, which would help users to spot the table format. If the columns stay centered they will be too close, which can be not too readable.